modify-variable.md

```diff
@@ -0,0 +1,43 @@
+# `@ModifyVariable`, cancellable
```

Comment:
This isn't cancellable, is it? At least, the examples are not.

Reply:
I definetly did not copy paste this from the inject, cancellable one :tiny_potato:

Reply:
https://www.d-e-f-i-n-i-t-e-l-y.com/